<!-- Simple stress test that launches a TF server and a few clients. -->

<launch>
  <node name="static_tf" pkg="tf2_ros" type="static_transform_publisher"
    args="1 2 3 0 0 0 map odom" output="screen" required="true" />

  <!-- Server -->
  <node name="simple_tf_buffer_server" pkg="simple_tf_buffer_server"
        type="server" args="--num_threads 5" output="screen" required="true" />

  <!-- Test node -->
  <test test-name="client_rostest" pkg="simple_tf_buffer_server"
        type="client_rostest" />

</launch>
